plugins/modules/nd_multi_cluster_connectivity.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_multi_cluster_connectivity
+version_added: "1.0.0"
+short_description: Manages cluster configurations on Cisco Nexus Dashboard.
+description:
+- This module allows for the management of clusters on Cisco Nexus Dashboard.
+- This module is only supported on ND v4.1 and later.
+author:
+- Shreyas Srish (@shrsr)
+options:
+  cluster_type:
+    description:
+    - The type of the cluster.
+    type: str
+    choices: [ nd, apic ]
+  cluster_hostname:
+    description:
+    - The hostname or IP address of the cluster.
+    type: str
+  cluster_username:
+    description:
+    - The username for authenticating with the cluster.
+    type: str
+  cluster_password:
+    description:
+    - The password for authenticating with the cluster.
+    - This value is not logged in the output.
+    type: str
+  cluster_login_domain:
+    description:
+    - The login domain for the cluster.
+    type: str
+  multi_cluster_login_domain:
+    description:
+    - The multi-cluster login domain.
+    type: str
+  fabric_name:
+    description:
+    - The name of the fabric to which the cluster belongs.
+    type: str
+  license_tier:
+    description:
+    - The license tier for the cluster.
+    type: str
+    choices: [ advantage, essentials, premier ]
+  features:
+    description:
+    - A list of features to enable on the cluster.
+    type: list
+    elements: str
+    choices: [ telemetry, orchestration ]
+  inband_epg:
+    description:
+    - The in-band EPG (Endpoint Group) for the cluster.
+    type: str
+  outband:
+    description:
+    - The out-of-band management configuration for the cluster.
+    type: str
+  orchestration:
```

Comment:
Do you need this option? It appears orchestration in the features list is used instead.